src/foraging_gui/Dialogs.py

```diff
@@ -1590,14 +1590,7 @@ def _Save(self):
                             input_voltages= self._extract_elements(LCM_voltage,current_frequency_ind)
                             input_voltages_unique=list(set(input_voltages))
                             for laser_tag in laser_tags:
-                                for laser_tag in laser_tags:
-                                    ItemsLaserPower=[]
-                                    for n in range(len(input_voltages_unique)):
-                                        current_voltage=input_voltages_unique[n]
-                                        laser_ind=[]
-                                        for k in range(len(input_voltages)):
-                                            if input_voltages[k]==current_voltage and LCM_Location_1[k]==f"Laser_{laser_tag}":
-                                                laser_ind.append(k)
+                                ItemsLaserPower=self._module_1(self,input_voltages_unique,input_voltages,LCM_Location_1,LCM_LaserPowerMeasured,laser_tag)
```

Comment:
use better function name than `_module_1`

Reply:
updated the name